airflow/providers/celery/executors/celery_kubernetes_executor.py

```diff
@@ -55,14 +55,21 @@ class CeleryKubernetesExecutor(LoggingMixin):
 
     callback_sink: BaseCallbackSink | None = None
 
-    KUBERNETES_QUEUE = conf.get("celery_kubernetes_executor", "kubernetes_queue")
+    @property
```

Comment:
We should not do it straight away, because we can reload the configuration - we would have to remove cached property after reload and I think that would be a bit too much of a coupling. But I will see what I can do in the final version (I am already reloading sensitive variables list in a similar way).